src/ImgixServiceProvider.php

```diff
@@ -13,6 +15,10 @@ public function boot(): void
         $this->publishes([
             __DIR__.'/../config/imgix.php' => config_path('imgix.php'),
         ], 'config');
+
+        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'imgix');
```

Comment:
The view should not only be loaded but also published so the user can override it.

Reply:
Fixed.